keras/metrics.py

```diff
@@ -91,6 +91,32 @@ def matthews_correlation(y_true, y_pred):
 
     return numerator / (denominator + K.epsilon())
 
+
+def fmeasure(y_true, y_pred, beta=1):
+    '''Approximate F-score as an overall loss for all the samples at once.
```

Comment:
This docstring will become part of the documentation, so it will need to be more informative. See the sklearn docs for the same feature for inspiration.